bouncer/src/repo/lib/datastructure/repo_vector3d.h

```diff
@@ -24,15 +24,19 @@
 #include <sstream>
 #include <cmath>
 
+// Undefine windef.h's wierdness
```

Comment:
Unfortunately not; the difference is that `NOMINMAX` can prevent windef.h making the defines in the first place, but repo_vector3d.h is often included after something that brings in windef.h, in which case it's too late.

The alternative is to find everywhere repo_vector3d.h is used, and move it to the top. I chose undef over this, because that would touch a load of unrelated files, and also mean the ordering of the headers would become significant without there being any indication that was the case, leading to a bunch of obscure errors if someone reorders them, adds a new header, includes repo_vector somewhere new, etc..

Reply:
ok fair enough. 

In which case can we please fix the spelling on `weird` 🙈

Reply:
🙈 fixed!